op-batcher/batcher/channel_builder_test.go

```diff
@@ -618,3 +632,75 @@ func TestFramePublished(t *testing.T) {
 	// Now the timeout will be 1000
 	require.Equal(t, uint64(1000), cb.timeout)
 }
+
+func TestChannelBuilder_InputBytes(t *testing.T) {
+	require := require.New(t)
+	rng := rand.New(rand.NewSource(time.Now().UnixNano()))
+	cb, _ := defaultChannelBuilderSetup(t)
+
+	require.Zero(cb.InputBytes())
+
+	var l int
+	for i := 0; i < 5; i++ {
+		block := newMiniL2Block(rng.Intn(32))
+		l += blockBatchRlpSize(t, block)
+
+		_, err := cb.AddBlock(block)
+		require.NoError(err)
+		require.Equal(cb.InputBytes(), l)
+	}
+}
+
+func TestChannelBuilder_OutputBytes(t *testing.T) {
+	require := require.New(t)
```

Comment:
You shouldn't need to do this, `require` is already imported from testify.

Reply:
See [docs](https://pkg.go.dev/github.com/stretchr/testify/require#New), this creates an assertion object `require` (that shadows the `require` package) bound to the `t *testing.T` so that assertions in this test don't need to always specify `t`. E.g. after this, we can write `require.NoError(SomeFunc())` instead of `require.NoError(t, SomeFunc())`.

See also the [`testify's README`](https://github.com/stretchr/testify#assert-package) section about `assert`, they recommend this pattern of shadowing the package name with `.New(t)` if multiple assertions are made in a test.

I recommend reading the testify go docs and readme, they have many hidden gems.